src/views/docs-view/server.ts

```diff
@@ -178,6 +181,12 @@ export function getStaticGenerationFunctions<
 					 * expected to have been run for remote content.
 					 */
 					[anchorLinks, { headings }],
+					/**
+					 * Remove the `<h1 />` from MDX, we'll render this outside
+					 * the MDX content area, integrating it into our layout
+					 * in various ways depending on the specific docs view used.
+					 */
+					remarkPluginRemoveH1,
```

Comment:
We now _always_ remove `h1` elements from MDX content for docs pages, as all docs page render `DocsPageHeading`, which contains an `h1`.

Note: The one exception is Packer plugins, but [Packer plugins has separate MDX processing](https://github.com/hashicorp/dev-portal/blob/fec683f2273e4df815a479856efe0a0902b04e07/src/views/packer-plugins/server.ts#L186), so we don't need to worry about that here. This is something we will untangle in the near future once Packer integrations launch: 🎟 [[cleanup] Remove docs page heading cruft once /packer/plugins is replaced by integrations](https://app.asana.com/0/1202097197789424/1204412156894157/f)